hazelcast/src/main/java/com/hazelcast/multimap/impl/MultiMapPartitionContainer.java

```diff
@@ -61,9 +61,20 @@ public MultiMapContainer getOrCreateMultiMapContainer(String name, boolean isAcc
         return container;
     }
 
-    public MultiMapContainer getMultiMapContainer(String name) {
+    /**
+     * Returns the {@link MultiMapContainer} with the given {@code name}
+     * if exists or {@code null otherwise}. Depending on the {@code isAccess}
+     * parameter this call updates the {@code lastAccessTime} field of the
+     * container.
+     *
+     * @param name     The name of the container to retrieve
+     * @param isAccess Indicates whether or not this call should be treated
+     *                 as an access
+     * @return the container or {@code null} if doesn't exist
+     */
+    public MultiMapContainer getMultiMapContainer(String name, boolean isAccess) {
```

Comment:
shouldn't this access info stored at record level? counting access as container access seems misleading.

Reply:
We don't have this granularity on Multimaps, at least not that i know of.

Reply:
just caught my attention, but yes we don't have it and it is out of scope for this PR.

Reply:
@ahmetmircik I agree. Changing the split-brain healing logic for the MultiMaps would need a considerable effort though and I'm not sure we can include it in 4.0. Created https://github.com/hazelcast/hazelcast/issues/16005 labeled with 4.0 for now.